app/controllers/api/v2/operatingsystems_controller.rb

```diff
@@ -90,6 +90,7 @@ def destroy
       def bootfiles
         Foreman::Deprecation.api_deprecation_warning("Bootfiles should be calculated per host")
 
+        action_permission
```

Comment:
What's this?

Reply:
Ops, I forgot that there. I'll fix it right away.